src/content/quickstart/openshift/ready_clusters.md

```diff
@@ -1,23 +1,13 @@
 ### Make your clusters ready for Submariner
 
-Submariner gateway nodes need to be able to accept traffic over ports 4500/UDP and 500/UDP
-when using IPSEC. In addition we use port 4800/UDP to encapsulate traffic from the worker nodes
-to the gateway nodes and ensuring that Pod IP addresses are preserved.
+Submariner gateway nodes need to be able to accept traffic over UDP ports (4500 and 500 by default) when using IPSEC.
+In addition we use port 4800/UDP to encapsulate traffic from the worker nodes to the gateway nodes and ensuring that Pod IP addresses are preserved.
 
 Additionally, the default OpenShift deployments don't allow assigning an elastic public IP
 to existing worker nodes, something that it's necessary at least on one end of the IPSEC connections.
 
-To handle all those details we provide a script that will prepare your AWS OpenShift deployment
-for Submariner, and will create an additional gateway node with an external IP.
+To handle all those details we provide a script
+[prep_for_subm.sh](https://github.com/submariner-io/submariner/blob/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh)
+that will prepare your AWS OpenShift deployment for Submariner, and will create an additional gateway node with an external IP.
 
-```bash
-
-curl https://raw.githubusercontent.com/submariner-io/submariner/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh -L -O
-chmod a+x ./prep_for_subm.sh
-
-./prep_for_subm.sh cluster-a     # respond yes when terraform asks
-./prep_for_subm.sh cluster-b      # respond yes when terraform asks
-
-```
-
-> **_INFO_** Please note that  **oc**, **aws-cli**, **terraform**, and **unzip** need to be installed before running the `prep_for_subm.sh` script.
+See instructions how to [make-your-clusters-ready-for-submariner](https://submariner.io/quickstart/openshift/ready_clusters/#make-your-clusters-ready-for-submariner).
```

Comment:
IMO, we can keep the bash commands here instead of a separate page as its hardly three or four steps.

Reply:
Duplicating instructions will require duplicating maintenance on both pages. @nyechiel what do you think ?

Reply:
I'm not saying to duplicate the instructions. Please take a look at the rendered page, it gives some details but does not give the instructions. 

![Screenshot from 2020-10-19 20-41-00](https://user-images.githubusercontent.com/6574656/96470136-96fde180-124b-11eb-9215-6fd5b4c2a407.png)

Now, if I click on the hyperlink, it repeats the same instructions (along with bash commands) and the rendering on the left pane shows a white line. So, my suggestion is instead of showing the bash instructions in a separate page, lets include them here itself.

![Screenshot from 2020-10-19 20-43-08](https://user-images.githubusercontent.com/6574656/96470294-c3b1f900-124b-11eb-9053-2c5552f8dc48.png)